src/stats/probs/mod.rs

```diff
@@ -403,7 +403,7 @@ impl From<NotNan<f64>> for LogProb {
 
 impl From<LogProb> for NotNan<f64> {
```

Comment:
Should probably also add a `TryFrom` impl here for people who want to acknowledge the fact that the conversion is fallible.